### ISSUE TITLE
Strings not highlighted anymore

### ISSUE THREAD

**Issue by lra** [cmur2/language-terraform#53]
I use a theme that's supposed to highlight strings:
![image](https://user-images.githubusercontent.com/122286/114763762-2c78ba80-9d31-11eb-8eb6-2b8ba498ba84.png)
https://github.com/atom/atom/tree/master/packages/one-dark-syntax

Up to today, it was highlighting strings in terraform too.
But not sure what changed today, strings in terraform are not highlighted anymore:
![Screen Shot 2021-04-14 at 2 54 57 PM](https://user-images.githubusercontent.com/122286/114763959-63e76700-9d31-11eb-9184-13351db06bc2.png)

Any idea how to highlight strings back?

**Comment by cmur2** [comment]
There are no recent updates to this package as seen in the version history: https://github.com/cmur2/language-terraform/releases

Which version of Atom and of the `language-terraform` package are you using? Maybe we can narrow down what changed :)

**Comment by lra** [comment]
Yes, that's why I'm asking for help, I don't know what happened =(

```
$ atom --version
Atom    : 1.56.0
Electron: 9.4.4
Chrome  : 83.0.4103.122
Node    : 12.14.1
```

![Screen Shot 2021-04-14 at 8 49 22 PM](https://user-images.githubusercontent.com/122286/114798249-e9373f80-9d62-11eb-8fcb-4434b625720b.png)

**Comment by craigwatson** [comment]
I'm using the versions of all components and the same Atom version, and I seem to have lost some of the secondary syntax highlighting for Terraform:

![image](https://user-images.githubusercontent.com/1749345/114884879-0a993980-9dfe-11eb-8e52-b74ded3eee06.png)

![image](https://user-images.githubusercontent.com/1749345/114885104-416f4f80-9dfe-11eb-87c6-afd7892aef6d.png)

**Comment by cmur2** [comment]
Thanks for both of your input! I will have a look at the problem, whether I can reproduce it and what might be causing it.

My first goto was https://atom.io/releases to check whether it mentions anything noteworthy regarding syntax highlighting changes... nothing obvious to me.

**Comment by cmur2** [comment]
I can reproduce the issue locally when updating my Atom from version 1.53.0 to 1.56.0:

![Auswahl_1460](https://user-images.githubusercontent.com/800933/114947000-920d9980-9e4c-11eb-8e2e-51e4a81ded67.png) ![Auswahl_1459](https://user-images.githubusercontent.com/800933/114946976-89b55e80-9e4c-11eb-879c-f3aeb8b31040.png)

Hmmm...

**Comment by cmur2** [comment]
@lra @craigwatson I'm happy to report that the syntax highlighting works again with no changes on current https://atom.io/nightly (1.58.0-nightly) - the comment https://github.com/atom/atom/issues/22168#issuecomment-817189907 made me try this. Would be great if you can confirm this in case you feel adventurous enough to try the nightly.

So for now I'd recommend downgrading to the last known good Atom version 1.55.0 if you consider the weird highlighting of language-terraform important enough + that's possible for you 🙂

**Comment by lra** [comment]
Not for me with nightly =(
![Screen Shot 2021-04-15 at 9 16 37 PM](https://user-images.githubusercontent.com/122286/114957940-0128c500-9e30-11eb-9ea9-40a93daa646e.png)
![Screen Shot 2021-04-15 at 9 17 18 PM](https://user-images.githubusercontent.com/122286/114957941-0128c500-9e30-11eb-83fe-cee5856faebc.png)

**Comment by craigwatson** [comment]
Also sadly not fixed for me with the latest nightly build:

![image](https://user-images.githubusercontent.com/1749345/115011996-30c6e400-9ea7-11eb-95a4-721a5a3db57f.png)

![image](https://user-images.githubusercontent.com/1749345/115012051-41775a00-9ea7-11eb-92e6-2768b81cd6d7.png)

**Comment by craigwatson** [comment]
Confirmed that rolling back to 1.55.0 fixes the problem though:

![image](https://user-images.githubusercontent.com/1749345/115012381-acc12c00-9ea7-11eb-930c-745822a17ec0.png)

![image](https://user-images.githubusercontent.com/1749345/115012421-b6e32a80-9ea7-11eb-9b10-653212a45dc2.png)

**Comment by cmur2** [comment]
Given that there is still activity and new reports coming in at https://github.com/atom/atom/issues/22168 I'd not spent further time troubleshooting for now since I realize that the Atom nightly build is a moving target -- it might behave differently every day depending on what upstream does.

Recommendation for now would be going back to 1.55.0 and wait what a release of 1.57.0 and/or 1.58.0 brings.

**Comment by Fran-Rg** [comment]
Release link: https://github.com/atom/atom/releases/tag/v1.55.0

**Comment by lra** [comment]
FTR, 1.57.0 just got released and the problem is still happening.

**Comment by cmur2** [comment]
> FTR, 1.57.0 just got released and the problem is still happening.

@lra looks like the problems still persist with Atom 1.57.0 also for other people, such as https://github.com/atom/atom/issues/22168#issuecomment-841309800

I'm still leaning towards waiting how https://github.com/atom/atom/issues/22168 develops (whether it's a theme issue or a language syntax issue) as it's not clear to me how to proceed. During that time downgrading to Atom 1.55.x is what I'd recommend :/
